### ISSUE TITLE
Play audio segments as soon as we move to next segment

### ISSUE THREAD

**Issue by kaustuk** [HumanSignal/label-studio#1680]
**Is your feature request related to a problem? Please describe.**
Consider the following scenario, 

An ASR system is giving a segment and we populate it in label studio. Now the person who is doing labeling needs to listen to the region and write text in textarea with is per region. To iterate over the region he would be using `Alt + .` to go to the next region. Doing this still it doesn't play the audio region automatically. So we need to click on the audio pane to actually play the audio region. Which slows down the labeling process.

The current process step looks something like this
1. We **select the region to play it**. And write the transcript in textarea.
2. After completing the first step **to move to the next region we press** `Alt + .` 
3. **Next segment is selected but the audio is not played** to actually hear the audio region, **we need to go to mouse move the cursor to the start of the region, and hit play**

Ideally, steps should be this
1. We **select the region to play it.** And write the transcript in textarea.
2. After completing the first step to move to the next region we press `Alt + .` . **The audio region started playing**

**TLDR**
**Want audio segment to be played automatically as soon as we press `Alt + .`**

**Comment by kaustuk** [comment]
Hi @makseq, thanks for the reply and sorry for the delay in reply from my side. Yeah, that makes sense to have this feature configurable and it's the user's call if they want to enable it or not. 

Just to be clear **LSF settings** is this panel, right?

<img width="596" alt="Screenshot 2021-11-11 at 14 21 58" src="https://user-images.githubusercontent.com/16715570/141267598-7047f980-0f20-4b84-8404-64bfc6c29a79.png">

**Comment by makseq** [comment]
@kaustuk Yes, LSF Settings is what we can see on this screenshot. "Autoplay next region" makes sense.

**Comment by kaustuk** [comment]
Hi @makseq, I have created the PR with this feature implemented. I tested it manually and it's working as expected. 

I haven't added any unit test case. Since I am not completely aware of the codebase and newbie in react, Can you provide some pointers around which test case I should refer to write testcase for this feature?

**Comment by kaustuk** [comment]
Hi @makseq , it's been a long time I have created PR with this feature, any approx timeline when PR would get reviewed or this feature would get added?

**Comment by makseq** [comment]
@kaustuk Sorry for this long waiting, I'll force our frontend team to merge it.

**Comment by kaustuk** [comment]
Hi @makseq, sorry to bother you. Just wanted to check the status on this. Any thought by when this feature/PR would get reviewed?

**Comment by makseq** [comment]
@kaustuk Thank you for pinging, I've asked to review your PR, then we will merge it.

**Comment by hlomzik** [comment]
Hey! Thanks for contribution! We already have this implemented in new Audio along with other convenient improvements! Please, check it, it's enabled by default. https://labelstud.io/tags/audio
Thus I have to close this issue.
